code/modules/antagonists/heretic/knowledge/knock_lore.dm

```diff
@@ -0,0 +1,242 @@
+	cost = 1
+	route = PATH_KNOCK
+
+/datum/heretic_knowledge/limited_amount/riteofpassage // item that creates 3 max at a time heretic only barriers, probably should limit to 1 only, holy people can also pass
+	name = "Rite Of Passage"
+	desc = "Allows you to transmute a white crayon, a wooden plank, and a multitool to create Consecrated Lintel. \
+		It can materialize a barricade at range, that only you and people resistant to magic can pass. 3 uses."
+	gain_text = "This is the skull of a door through which power has passed."
+	required_atoms = list(
+		/obj/item/toy/crayon/white = 1,
+		/obj/item/stack/sheet/mineral/wood = 1,
+		/obj/item/multitool = 1,
+	)
+	result_atoms = list(/obj/item/heretic_lintel)
+	next_knowledge = list(/datum/heretic_knowledge/mark/knock_mark)
+	cost = 1
+	route = PATH_KNOCK
+
+/datum/heretic_knowledge/mark/knock_mark
+	name = "Mark of Knock"
+	desc = "Your Mansus Grasp now applies the Mark of Knock. Attack a marked person to corrupt access on \
+		all of their keycards for the duration of the mark. \
+		This will make it so that they have no access whatsoever, and even public access doors will deny their passage."
+	gain_text = "When the moon's face was wounded, these words appeared."
+	next_knowledge = list(/datum/heretic_knowledge/knowledge_ritual/knock)
+	route = PATH_KNOCK
+	mark_type = /datum/status_effect/eldritch/knock
+
+/datum/heretic_knowledge/knowledge_ritual/knock
+	next_knowledge = list(/datum/heretic_knowledge/spell/burglar_finesse)
+	route = PATH_KNOCK
+
+/datum/heretic_knowledge/spell/burglar_finesse
+	name = "Burglar's Finesse"
+	desc = "Grants you Burglar's Finesse, a single-target spell \
+		that puts a random item from the victims storage into your hand."
+	gain_text = "She in her many echoing voices told me, that burglars, locksmiths, and so on all share one thing;"
+	next_knowledge = list(
+		/datum/heretic_knowledge/spell/apetra_vulnera,
+		/datum/heretic_knowledge/blade_upgrade/flesh/knock,
+	)
+	spell_to_add = /datum/action/cooldown/spell/pointed/burglar_finesse
+	cost = 2
+	route = PATH_KNOCK
+
+/datum/heretic_knowledge/spell/apetra_vulnera
+	name = "Apetra Vulnera"
+	desc = "Grants you Apetra Vulnera, a spell \
+		that causes heavy bleeding on all bodyparts of the victim that have more than 15 brute."
+	gain_text = "To open certain Ways, one must first open oneself."
+	next_knowledge = list(
+	/datum/heretic_knowledge/spell/blood_siphon,
+	/datum/heretic_knowledge/void_cloak,
+	)
+	spell_to_add = /datum/action/cooldown/spell/pointed/apetra_vulnera
+	cost = 1
+	route = PATH_SIDE
+
+/datum/heretic_knowledge/blade_upgrade/flesh/knock //basically a chance-based weeping avulsion version of the former
+	name = "Opening Blade"
+	desc = "Your blade has a chance to cause a weeping avulsion on attack."
+	gain_text = "They open, remove seals."
+	next_knowledge = list(/datum/heretic_knowledge/spell/caretaker_refuge)
+	route = PATH_KNOCK
+	wound_type = /datum/wound/slash/flesh/critical
+	var/chance = 35
```

Comment:
Auto doc

Reply:
knock is the aspect that opens

Reply:
adding to this, in cultist sim lore getting a grevious wound is part of getting knock lore

Reply:
a weeping avulsion is grievous